package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@conversationlearner/models": "0.197.0",
     "@conversationlearner/webchat": "0.145.0",
+    "@types/fast-json-stable-stringify": "^2.0.0",
```

Comment:
There should be a corresponding change in the `package-lock.json`.  You builds seem to be failure because of this.

Reply:
Crossed paths in the PR. Now listed in the package.json dependency list

Reply:
Hmm. I'm still not seeing the associated change in the `package-lock.json`.

`@types/*` dependencies should be `devDependencies`

Also it looks like you are using `fast-json-stable-stringify` so it should also be a dependency.